.appveyor.yml

```diff
@@ -7,36 +7,59 @@ clone_depth: 5
+  QT_DOWNLOAD_HASH: 'D4D35B8112302B67E5610A03421BB3E43FE13F14D9A5F637C22AE60DCEC0E0F5'
+  QT_LOCAL_PATH: 'C:\Qt5.9.7_ssl_x64_static_vs2017'
+  LIBPROTOBUFLITE_PATH: 'c:\tools\vcpkg\installed\x64-windows-static\lib\libprotobuf-lite.lib'
+  LIBPROTOBUFLITE_DESTINATION_PATH: 'c:\tools\vcpkg\installed\x64-windows-static\lib\libprotobuf-lite.lib'
 cache:
-- C:\tools\vcpkg\installed -> .appveyor.yml
+- C:\tools\vcpkg\installed
 - C:\Users\appveyor\clcache -> .appveyor.yml, build_msvc\**, **\Makefile.am, **\*.vcxproj.in
+- C:\Qt5.9.7_ssl_x64_static_vs2017
 install:
 - cmd: pip install --quiet git+https://github.com/frerich/clcache.git@v4.2.0
 # Disable zmq test for now since python zmq library on Windows would cause Access violation sometimes.
 # - cmd: pip install zmq
 - cmd: echo set(VCPKG_BUILD_TYPE release) >> C:\tools\vcpkg\triplets\%PLATFORM%-windows-static.cmake
 - cmd: vcpkg remove --outdated --recurse
 - cmd: vcpkg install --triplet %PLATFORM%-windows-static %PACKAGES% > NUL
+# The libprotobuf-lite.lib has duplicate symbols to libprotobuf.lib and breaks bitcoin-qt and test_bitcoin-qt when ENABLE_BIP70 is set.
+- ps: |
+      if(Test-Path -Path ($env:LIBPROTOBUFLITE_PATH)) {
+         Write-Host "Moving libprotobuf-lite library to manual link directory.";
+         Move-Item -Path $env:LIBPROTOBUFLITE_PATH -Destination $env:LIBPROTOBUFLITE_DESTINATION_PATH
+      }
 before_build:
 - ps:  clcache -M 536870912
-- cmd: python build_msvc\msvc-autogen.py
-- ps:  $files = (Get-ChildItem -Recurse | where {$_.extension -eq ".vcxproj"}).FullName
-- ps:  for (${i} = 0; ${i} -lt ${files}.length; ${i}++) {
-           ${content} = (Get-Content ${files}[${i}]);
-           ${content} = ${content}.Replace("</RuntimeLibrary>", "</RuntimeLibrary><DebugInformationFormat>None</DebugInformationFormat>");
-           ${content} = ${content}.Replace("<WholeProgramOptimization>true", "<WholeProgramOptimization>false");
-           Set-Content ${files}[${i}] ${content};
-       }
+- ps: |
+      if(!(Test-Path -Path ($env:QT_LOCAL_PATH))) {
+        Write-Host "Downloading Qt binaries.";
+        Invoke-WebRequest -Uri $env:QT_DOWNLOAD_URL -Out qtdownload.zip;
+        Write-Host "Qt binaries successfully downloaded, checking hash against $env:QT_DOWNLOAD_HASH...";
+        if((Get-FileHash qtdownload.zip).Hash -eq $env:QT_DOWNLOAD_HASH) {
+          Expand-Archive qtdownload.zip -DestinationPath $env:QT_LOCAL_PATH;
+          Write-Host "Qt binary download matched the expected hash.";
+        }
+        else {
+          Write-Host "ERROR: Qt binary download did not match the expected hash.";
+          Exit-AppveyorBuild;
+        }
+      }
+      else {
+         Write-Host "Qt binaries already present.";
+      }
+- cmd: python build_msvc\msvc-autogen.py -qtdir "C:\\Qt5.9.7_ssl_x64_static_vs2017" -protocpath "C:\\Tools\\vcpkg\\installed\\x64-windows-static\\tools\\protobuf\\protoc.exe"
```

Comment:
If you change the python code not to use regular expression. It could be single slash here.